packages/nextcloud_test/docker/Dockerfile

```diff
@@ -21,13 +21,18 @@ RUN (sh /entrypoint.sh php -S 0.0.0.0:80 &) && \
     for user in admin user1 user2; do curl -u "$user:$user" -H "ocs-apirequest: true" -s -o /dev/null http://localhost/ocs/v2.php/cloud/user; done
 
 COPY static /usr/src/nextcloud/static
+COPY assets/Recipes /usr/src/nextcloud/data/user1/files/Recipes
```

Comment:
Not necessary for the unit tests, but for general testing it would be nice if all users had the recipes (so admin and user2 as well).